### PR TITLE
Add a way to filter topics by status

### PR DESCRIPTION
**Pull request by riccardo-forina**
image

image

image

### CODE REVIEW

File: ui/api/topics/actions.ts [hunk 1]
```diff
@@ -38,6 +40,7 @@ export async function getTopics(
         "name,status,visibility,numPartitions,totalLeaderLogBytes,consumerGroups",
       "filter[id]": params.id ? `eq,${params.id}` : undefined,
       "filter[name]": params.name ? `like,*${params.name}*` : undefined,
+      "filter[status]": params.status,
```
Comment (MikeEdgar) [inline ui/api/topics/actions.ts:43]: In this case where there is no "operator" in the filter like `eq` or `in`, the API defaults to `eq`. If you ever want to search for multiple status values, the `in` could be used.

**Review by MikeEdgar — APPROVED** [review]

File: ui/api/topics/actions.ts (hunk 1 above)
Comment (riccardo-forina) [reply ui/api/topics/actions.ts:43]: Ah it wasn't clear from the openapi doc, I thought it supported a single value only

Comment (MikeEdgar) [reply ui/api/topics/actions.ts:43]: Yeah, I missed docs for many of the `filter` parameters. They should all follow that pattern. I'll see if there's a nice way to document that without too much repetition.

Comment (riccardo-forina) [reply ui/api/topics/actions.ts]: Done, updated screens in the PR description to match the new option

**Pull request merged**